### ISSUE TITLE
Is MongooseIM Docker Project Dead?

### ISSUE THREAD

**Issue by DaniloMurbach** [esl/mongooseim-docker#50]
HI, this looks like a great project how to build and run the mongooseIM as containers, I'm working to build them as a docker-compose which you can simply deploy running docker compose up or building a stack in docker Swarm.

I'm trying to add the database as well, but then I cannot see any reference of the files or details where de back end rest is or what are the steps for mongooseIM init the database.

So basically my question is..

 Is this docker documentation a project that could run actually as a cluster and with all the features that MongooseIM offers ?

**Comment by chrzaszcz** [comment]
Hi, this project is not dead, and it is used to automatically build all the latest Docker images, see https://hub.docker.com/repository/docker/mongooseim/mongooseim/general

You can either use these images directly, or use Kubenetes and Helm: https://artifacthub.io/packages/helm/mongoose/mongooseim

**Comment by DaniloMurbach** [comment]
Thanks a Lot, ok. Paweł Chrząszcz 

I will follow that

**Comment by arcusfelis** [comment]
SQL files are there:
- https://github.com/esl/MongooseIM/tree/master/priv
you would have to apply them in your database. I.e. for PgSQL file is pg.sql, for MySQL it is mysql.sql.
You would have to setup DB from the official docker image for the specific database.

You could provide mongooseim config as a volume if you are using the dockerfile.

Recommended way is to use Helm if you are doing something production ready https://github.com/esl/MongooseHelm
